crates/turborepo-ui/src/tui/app.rs

```diff
@@ -226,14 +397,17 @@ fn update(
 ) -> Result<Option<mpsc::SyncSender<()>>, Error> {
     match event {
         Event::StartTask { task } => {
-            app.table.start_task(&task)?;
-            app.started_tasks.push(task);
+            app.start_task(&task)?;
+            app.tasks_by_status
+                .groups_as_task_names()
+                .running
+                .push(task);
```

Comment:
This isn't doing what I think you mean for it to do.

`groups_as_task_names` is constructing a new structure that is getting the new task in `running`, but that structure immediately goes out of scope and is dropped.

Reply:
Yikes, definitely not.

Reply:
Removed entirely.